src/business_logic/execution/objects.rs

```diff
@@ -38,7 +39,7 @@ pub struct CallInfo {
     pub(crate) caller_address: Address,
     pub(crate) call_type: Option<CallType>,
     pub(crate) contract_address: Address,
-    pub(crate) class_hash: Option<Felt>,
+    pub(crate) class_hash: [u8; 32],
```

Comment:
In the Python implementation this is an option
Are you sure it is ok to remove it?

Reply:
i missed it, i remember that i put it as Option but an error appeared. I put it again as an option to match python implementation. thanks for pointing it out.